DRIVERS/Z001_SMB/z001_smb.c

```diff
@@ -127,7 +127,11 @@ static int G_instance=0;
  */
 static void men16z001_pause( unsigned int amount )
 {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,0)
+	WRITE_ONCE(current->__state, TASK_STOPPED);
```

Comment:
Is TASK_STOPPED the right value here, or should it be "WRITE_ONCE(current->__state, TASK_INTERRUPTIBLE);"? The description of schedule_timeout() at https://www.kernel.org/doc/html/v5.15/driver-api/basics.html describes no behavior for TASK_STOPPED.